re/.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
```

Comment:
We can remove this file, and port the esy build steps to the root `.travis.yml`

Reply:
Do we need travis if we're only targeting windows?

Reply:
No, good point, let's remove the `travis.yml` and simplify the pipeline 👍